governance/xc-admin/packages/xc-admin-frontend/components/tabs/MinPublishers.tsx

```diff
@@ -1,20 +1,235 @@
+    anchorWallet as Wallet
+  )
+  const { rawConfig, dataIsLoading, connection } = usePythContext()
+  const { connected } = useWallet()
+  const [pythProgramClient, setPythProgramClient] =
+    useState<Program<PythOracle>>()
+
+  const openModal = () => {
+    setIsModalOpen(true)
+  }
+
+  const closeModal = () => {
+    setIsModalOpen(false)
+  }
+
+  const handleEditButtonClick = () => {
+    const nextState = !editable
+    if (nextState) {
+      const newColumns = [
+        ...defaultColumns,
+        columnHelper.accessor('newMinPublishers', {
+          cell: (info) => info.getValue(),
+          header: () => <span>New Min Publishers</span>,
+        }),
+      ]
+      setColumns(newColumns)
+    } else {
+      if (
+        minPublishersChanges &&
+        Object.keys(minPublishersChanges).length > 0
+      ) {
+        openModal()
+        setMinPublishersChanges(minPublishersChanges)
+      } else {
+        setColumns(defaultColumns)
+      }
+    }
+
+    setEditable(nextState)
+  }
+
+  const handleEditMinPublishers = (
+    e: any,
+    symbol: string,
+    prevMinPublishers: number
+  ) => {
+    const newMinPublishers = Number(e.target.textContent)
+    if (prevMinPublishers !== newMinPublishers) {
+      setMinPublishersChanges({
+        ...minPublishersChanges,
+        [symbol]: {
+          prev: prevMinPublishers,
+          new: newMinPublishers,
+        },
+      })
+    }
+  }
+
+  useEffect(() => {
+    if (!dataIsLoading && rawConfig) {
+      const minPublishersData: MinPublishersProps[] = []
+      rawConfig.mappingAccounts
+        .sort(
+          (mapping1, mapping2) =>
+            mapping2.products.length - mapping1.products.length
+        )[0]
+        .products.map((product) =>
+          product.priceAccounts.map((priceAccount) => {
+            minPublishersData.push({
+              symbol: product.metadata.symbol,
+              minPublishers: priceAccount.minPub,
+            })
+          })
+        )
+      setData(minPublishersData)
+    }
+  }, [setData, rawConfig, dataIsLoading])
+
+  const table = useReactTable({
+    data,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+  })
+
+  const handleSendProposalButtonClick = async () => {
+    if (pythProgramClient && minPublishersChanges) {
+      const instructions: TransactionInstruction[] = []
+      Object.keys(minPublishersChanges).forEach((symbol) => {
+        const { prev, new: newMinPublishers } = minPublishersChanges[symbol]
+        const priceAccountPubkey = rawConfig.mappingAccounts
+          .sort(
+            (mapping1, mapping2) =>
+              mapping2.products.length - mapping1.products.length
+          )[0]
+          .products.find((product) => product.metadata.symbol === symbol)!
+          .priceAccounts.find(
+            (priceAccount) => priceAccount.minPub === prev
+          )!.address
+
+        pythProgramClient.methods
+          .setMinPub(newMinPublishers, [0, 0, 0])
```

Comment:
You need to make `fundingAccount : squads?.getAuthorityPDA(
            SECURITY_MULTISIG[getMultisigCluster(cluster)],
            1
          )` otherwise the multisig won't sign the transaction